src/main.ts

```diff
@@ -2,9 +2,13 @@ import "./main.css";
 import * as wt from "../lib/wtrace";
 import * as test from "./test-scenes";
 
-var canvas: HTMLCanvasElement;
-var fpsElement: HTMLTextAreaElement;
-var fpsDisplayTimer: number;
+// UI elements
```

Comment:
Why are these global variable ?

Reply:
just to easily access them at the test scene if needed, and to keep the main function easier to read